Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/Competition/BlueLeftAutonomousCompetition.java

```diff
@@ -0,0 +1,53 @@
+package org.firstinspires.ftc.team6220_PowerPlay.Competition;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+
+import org.firstinspires.ftc.robotcore.external.navigation.AngleUnit;
+import org.firstinspires.ftc.robotcore.external.navigation.AxesOrder;
+import org.firstinspires.ftc.robotcore.external.navigation.AxesReference;
+import org.firstinspires.ftc.team6220_PowerPlay.AprilTagDetect;
+import org.firstinspires.ftc.team6220_PowerPlay.Constants;
+
+@Autonomous(name = "BlueLeft", group = "Competition")
+public class BlueLeftAutonomousCompetition extends AprilTagDetect {
+
+    @Override
+    public void runOpMode() throws InterruptedException {
+        initialize();
+
+        servoGrabber.setPosition(0.45);
+        driveTurntable(1.0, Constants.TURNTABLE_DEFAULT_POSITION);
+        int signal = detectAprilTag();
+
+        IMUOriginalAngles = imu.getAngularOrientation(AxesReference.INTRINSIC, AxesOrder.ZYX, AngleUnit.DEGREES);
+        driveWithIMU(0.25, 0.0, 0.0);
+        sleep(1688);
```

Comment:
Where did you get this number?

Reply:
We found that our robot moves 27 inches in 1.688 seconds